pkg/cmd/analyzer/analyzer.go

```diff
@@ -0,0 +1,150 @@
+package analyzer
```

Comment:
WDYT of keeping this in the `cmd/analyzer/analyzer.go` package/file and creating a `cmd/analyzer/main/main.go` one instead which has the code that only calls the exported function? 

I think that, conceptually, I'd prefer the busybox importing "internal" packages in the `cmd` root folder then the "frontend" code from the CLI/cmd being exported as a lib in the `pkg` root 🤔

Reply:
Sure, that sounds reasonable!

Reply:
Cool, I think you'll just need to change the main file path here for building/running
https://github.com/livepeer/livepeer-data/blob/5b75a623cbb307520e42001663532c1572f6b46b/cmd/analyzer/Makefile#L10-L14